docs/source/eventing/examples/mqtt_publish.rst

```diff
@@ -0,0 +1,43 @@
+.. |yes| image:: ../../images/yes.png
+.. |no| image:: ../../images/no.png
+
+.. role:: underline
+   :class: underline
+
+.. versionadded:: nightly
+
+Publish mqtt message  on contact trigger
+==================================
```

Comment:
The equal signs should be as long as the title. Also, you have a bogus space in the title.